src/resources/experiments.ts

```diff
@@ -283,8 +284,32 @@ class ScoringHelper {
       .filter((f) => f.scorer.type === "local")
       .map(async (f) => {
         const scorer = f.scorer as LocalScorer;
-        const score = await scorer.scoreFn({ input, output, expected });
-        scores[f.registration.key_name] = score;
+
+        try {
+          scores[f.registration.key_name] = await scorer.scoreFn({
+            input,
+            output,
+            expected,
+          });
+        } catch (err) {
+          console.error(
+            `\nFailed to locally run score ${f.name.toLowerCase()}.`,
+            "\nNote: This error will be displayed in the dashboard. All other scoring will be preserved and displayed accordingly.",
+            "\nError received:",
+            err,
+            "\nFailing item object for reference:",
+            {
+              input,
+              expected,
+              output,
+            },
+            "\n\n",
```

Comment:
Let's not add the extra newlines. People might want to set up "grep" rules in the CLI runner to parse out errors. 
The entire message should be one line, and the error stack trace on the second line.

Reply:
Does the object print out correctly? I'm leaning towards not including the dataset record, since they can see that in the UI anyways.

Reply:
Removed the entire dataset record from this print in commit https://github.com/HammingHQ/evals-ts/pull/46/commits/4deae898c4676c00fd49e91f2cf34b7c2a47ce79.